wildfly-ai/subsystem/src/main/java/org/wildfly/extension/ai/model/chat/OpenAIChatModelProviderServiceConfigurator.java

```diff
@@ -82,6 +87,7 @@ public ChatLanguageModel get() {
             }
         };
         return CapabilityServiceInstaller.builder(CHAT_MODEL_PROVIDER_CAPABILITY, factory)
+                .onStart(registry.add(context.getCurrentAddressValue()))
```

Comment:
`ValueRegistry<K, V>` is basically an API for interacting with a `Map<K, AtomicReference<V>>`.
Typically, we also want to ensure that this map entry is removed on service stop.
e.g.
```java
.onStop(model -> registry.remove(context.getCurrentAddressValue());